torchtext/experimental/datasets/language_modeling.py

```diff
@@ -6,10 +6,12 @@
 
 
 def build_vocab(data, transforms):
```

Comment:
If we see a pattern to use `build_vocab` with `transform`, should we have a function in `experimental/functional.py`?

Reply:
Ideally these kind of functions are easy and succinct to write. What if we want another argument (say ngrams) etc.? It's much easier for the user to express this in small code snippets.